pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -358,45 +357,46 @@ func (p *provider) CollectLogs(dir string, nodes []nodes.Node) error {
 // Info returns the provider info.
 // The info is cached on the first time of the execution.
 func (p *provider) Info() (*providers.ProviderInfo, error) {
+	var err error
 	if p.info == nil {
-		p.info = info(p.logger)
+		p.info, err = info(p.logger)
 	}
-	return p.info, nil
+	return p.info, err
 }
 
-func info(logger log.Logger) *providers.ProviderInfo {
-	euid := os.Geteuid()
-	info := &providers.ProviderInfo{
-		Rootless: euid != 0,
-	}
-	if _, err := os.Stat("/sys/fs/cgroup/cgroup.controllers"); err == nil {
```

Comment:
maybe, we just need to skip this check if we can detect is a remote podman

```
if !podman_remote(U && _, err := os.Stat("/sys/fs/cgroup/cgroup.controllers"); err == nil {
```

Reply:
let me summon the expert, 
@vrothberg can I know from the podman cli if is running remote or local?

Reply:
Not yet but I opened https://github.com/containers/podman/issues/10289. @baude, do you know a workaround?

Reply:
@afrittoli can we try with this approach?
https://github.com/containers/podman/issues/10289#issuecomment-838302394
something like
```go
// podman_remote returns true if podman is using a remote client
// xref: https://github.com/containers/podman/issues/10289
func podman_remote() bool{
  ... check https://github.com/containers/podman/issues/10289#issuecomment-838302394 ...
}
```